CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
+### New Features
+- Public preview version of a new set of APIs to simplify the experience using IoT Plug and Play:
+  - az_iot_pnp_client_init
```

Comment:
nit: not sure whether I would rather see this here and elsewhere
```suggestion
  - az_iot_pnp_client_init()
```
i guess I'll let you make the call